### PR TITLE
PA-2544 "Last Updated By" on detail view

### PR DESCRIPTION
**Pull request by agalbrai**
image

### CODE REVIEW

**Review by jdtoombs — APPROVED** [review]

File: frontend/src/features/properties/components/LastUpdatedBy.tsx [hunk 1]
```diff
@@ -14,35 +14,43 @@ const UpdateText = styled.p`
...
-  color: #069;
+  color: #18548c;
   text-decoration: underline;
 `;
 
 /**
  * Display formatted last update user name, email, and date/time.
  */
-const LastUpdatedBy: React.FunctionComponent<ILastUpdatedByProps> = ({
+const LastUpdatedBy: React.FC<ILastUpdatedByProps> = ({
```
Comment (jdtoombs) [inline frontend/src/features/properties/components/LastUpdatedBy.tsx:28]: 😎

Comment (agalbrai) [reply frontend/src/features/properties/components/LastUpdatedBy.tsx:28]: I think that was an accident lol, I couldn't figure out what an error meant so I was just trying stuff

**Review by Fosol — APPROVED** [review]

File: frontend/src/features/properties/components/LastUpdatedBy.tsx [hunk 2]
```diff
@@ -14,35 +14,43 @@ const UpdateText = styled.p`
...
-      </TooltipWrapper>
-      &nbsp;
-      {formatApiDateTime(updatedOn ?? createdOn)}
-    </UpdateText>
-  ) : null;
+  if (updatedOn) {
+    return (
+      <UpdateText>
+        Last Updated By:&nbsp;
+        <TooltipWrapper toolTipId="lastUpdatedEmail" toolTip={updatedByEmail}>
```
Comment (devinleighsmith) [inline frontend/src/features/properties/components/LastUpdatedBy.tsx:38]: If I remember correctly, part of the reason for the 'unknown' is that we don't have a firm requirement on users in the system having an email associated, or even a name. Those are both optional fields.

Comment (agalbrai) [reply frontend/src/features/properties/components/LastUpdatedBy.tsx:38]: Ah, I gotcha. I'll put that back

**Pull request merged**